libsolidity/interface/CompilerStack.cpp

```diff
@@ -525,10 +529,28 @@ bool CompilerStack::compile()
 						m_errorReporter.error(_error.errorId(), _error.type(), SourceLocation(), _error.what());
 						return false;
 					}
-					if (m_generateIR || m_generateEwasm)
-						generateIR(*contract);
-					if (m_generateEwasm)
-						generateEwasm(*contract);
+					catch (UnimplementedFeatureError const& _unimplementedError)
+					{
+						if (
+							SourceLocation const* sourceLocation =
+							boost::get_error_info<langutil::errinfo_sourceLocation>(_unimplementedError)
```

Comment:
```suggestion
								boost::get_error_info<langutil::errinfo_sourceLocation>(_unimplementedError)
```
?

Reply:
it's a list, and all elements of the list have the same indentation

Reply:
What's a list? I meant the line after `=`, shouldn't it have one more indentation level?

Reply:
Operators do not affect indentation level, only parentheses do.

Reply:
I'm almost sure I've seen
```
auto x =
    \t something
```
before, but ok.